### PR TITLE
Edit pass: mixed-reality articles

### PR DESCRIPTION
**Pull request by jborsecnik**
Please see https://github.com/MicrosoftDocs/mixed-reality/pull/1918#issuecomment-562290559.

### CODE REVIEW

**Comment by jborsecnik** [comment]
Edit pass in progress. Please don’t review yet.

**Comment by jborsecnik** [comment]
Hi @NoelCross, @kegodin – Please review the copy edit to these articles, address the following questions, and let me know if I’ve changed the technical meaning anywhere. This is for Azure DevOps work request [1592807](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1592807).

Comments:

From **case-study-spacial-sound-design-for-holotour.md**:
-	DOCs articles are not supposed to have bylines. Do we have to retain the info about Jason Syltebo?
- What is “beats” here? This seems like too casual/slang. 
   *The journey though HoloTour contains certain beats where your tour guide, aided by holograms, explores a topic in-depth.…*

From **spacial-sound.md**
- Articles are supposed to get a "Next steps" section at the end. Can you suggest "Next Steps" for this article?

Fromn **input-in-directx.md**
- I’m not sure what this means exactly or how to make it clear; can you help revise?
   *Make use of the data **however applicable** to your app scenario.*

From **spacial-sound-design.md**
- The example .wav sounds don't play from the links. Do you have to download them? If so, is there a better/more convenient way to present them?

From **voice-input-in-directx.md**
- Those “try” instances in the upper-left corner of a couple of the code blocks look odd to me. Do they belong there?<br/>
image

- I had to shorten the following subheads, as H2s are supposed to not wrap to 3 lines in the on-page TOC. This was a bit awkward—if you have better ideas, or a way to shorten the first one further, let me know:
   - *Use SpeechRecognizer for continuous recognition of voice commands* -> *Use SpeechRecognizer for continuous speech recognition*
   - *Use dictation for one-shot recognition of speech phrases and sentences* -> *Use dictation for phrase recognition*

**TOC matters**
- **case-study-spatial-sound-design-for-holotour.md** lacks a TOC entry. Can you provide one and show where it should go?
- **directx-development-overview** - The TOC item is “**Native** development overview”—is that correct?

Thanks, jeff b

**Comment by kegodin** [comment]
Hi @jborsecnik 

Thanks for the review! Here are replies to your questions:

From case-study-spacial-sound-design-for-holotour.md:

- DOCs articles are not supposed to have bylines. Do we have to retain the info about Jason Syltebo?
_No, we can remove the byline. Could you also remove the byline from case-study-using-spatial-sound-in-roboraid.md?_
- What is “beats” here? This seems like too casual/slang.
_It's jargon in the storytelling industries (movies, tv, video games)._
 
From spacial-sound.md

- Articles are supposed to get a "Next steps" section at the end. Can you suggest "Next Steps" for this article?
_You could link to spatial-sound-in-unity.md and spatial-sound-design.md_

From input-in-directx.md

-I’m not sure what this means exactly or how to make it clear; can you help revise?
  - "Make use of the data however applicable to your app scenario."
_The sentence looks redundant. I suggest removing it._

From spacial-sound-design.md

- The example .wav sounds don't play from the links. Do you have to download them? If so, is there a better/more convenient way to present them?
_The user has to download them. Unfortunately the doc site has a video player (https://docs.microsoft.com/en-us/contribute/markdown-reference#videos) but not an audio player. I haven't been able to find any other examples of audio on the docs site._

From voice-input-in-directx.md

- Those “try” instances in the upper-left corner of a couple of the code blocks look odd to me. Do they belong there?

_The 'try' keyword is valid syntax here. It looks strange because it's not indented like the line that follows it. They should be indented the same amount as the following line, but looking at all blocks on that page it seems they've enforced a style where the first line of a code block isn't indented._

- I had to shorten the following subheads, as H2s are supposed to not wrap to 3 lines in the on-page TOC. This was a bit awkward—if you have better ideas, or a way to shorten the first one further, let me know:

  - Use SpeechRecognizer for continuous recognition of voice commands -> Use SpeechRecognizer for continuous speech recognition
_It could be shortened to 'Continuous recognition'_
  - Use dictation for one-shot recognition of speech phrases and sentences -> Use dictation for phrase recognition
_It could be shortened to 'One-shot recognition'_


TOC matters

- case-study-spatial-sound-design-for-holotour.md lacks a TOC entry. Can you provide one and show where it should go?
_Under Design -> UX Elements -> Style replace 'Spatial sound design' with a TOC subsection called 'Spatial sound', and move spatial-sound-design.md under it, and put the holotour and roboraid case studies under it._

- directx-development-overview - The TOC item is “Native development overview”—is that correct?
_It's under a TOC subsection called 'native', so the H1 on this page should be changed from 'Directx development overview' to 'Native development overview'.

File: mixed-reality-docs/case-study-spatial-sound-design-for-holotour.md [hunk 1]
```diff
@@ -10,57 +10,57 @@ keywords: Windows Mixed Reality, HoloLens, HoloTour, spatial sound, case study
 
 
 
-# Case study - Spatial sound design for HoloTour
+# Case study: Spatial sound design for HoloTour
 
-To create a truly immersive 3D virtual tour for Microsoft HoloLens, the panoramic videos and holographic scenery are only part of the formula. Audio designer Jason Syltebo talks about how sound was captured and processed to make you feel like you're actually in each of the locations in HoloTour.
+Panoramic videos and holographic scenery are only part of the formula for a truly immersive virtual Microsoft HoloLens tour. In this article, audio designer Jason Syltebo describes how sound was used to make you feel like you're actually in each HoloTour location.
```
Comment (kegodin) [inline mixed-reality-docs/case-study-spatial-sound-design-for-holotour.md]: Change "In this article, audio designer" to "This article"

File: mixed-reality-docs/spatial-sound-design.md [hunk 2]
```diff
@@ -1,113 +1,109 @@
...
+### Which spatialization technology to use
+With headphones or the HoloLens speakers, use head-related transfer function (HRTF)-based spatialization technologies. These devices model the sound propagation around the head in the physical world. Even when a sound source is on the far side of one's head, sound propagates to the distant ear with some attenuation and delay. Speaker panning, in contrast, relies only on attenuation and applies total attenuation in the left ear when sounds are on the right side (and vice-versa). This technique can be uncomfortable for "normal hearing" listeners and inaccessible for listeners who have hearing impairment in one ear.
```
Comment (kegodin) [inline mixed-reality-docs/spatial-sound-design.md]: "These devices" -> "These technologies"

File: mixed-reality-docs/case-study-spatial-sound-design-for-holotour.md (hunk 1 above)
Comment (jborsecnik) [reply mixed-reality-docs/case-study-spatial-sound-design-for-holotour.md]: Thanks. I also removed the little bio blurb about Jason at the end.

**Comment by jborsecnik** [comment]
@kegodin, in removing the byline/author info from **case-study-using-spatial-sound-in-roboraid.md** (three instances), as you requested, I noticed that this link is not working:

*(For more information about creating an audio asset with the Doppler effect built in, see [100 Whooshes in 2 Minutes](https://designingsound.org/2010/02/charles-deenen-special-100-whooshes-in-2-minutes/).)*

**Comment by kegodin** [comment]
@jborsecnik Nice find. Here's an updated link: http://designingsound.org/2010/02/26/charles-deenen-special-100-whooshes-in-2-minutes/

File: mixed-reality-docs/spatial-sound.md [hunk 3]
```diff
@@ -55,39 +55,38 @@ For more detailed examples of design using audio, see [sound design](spatial-sou
...
+Spatial audio is about more than direction. Other dimensions include occlusion, obstruction, reverb, portalling, and source modeling. Collectively these dimensions are referred to as *acoustics*. Without acoustics, spatialized sounds lack perceived distance.
 
+Acoustics treatments range from simple to very complex. You can use a simple reverb that's supported by any audio engine to push spatialized sounds into the environment of the listener. Acoustics systems such as [Project Acoustics](https://aka.ms/acoustics)  provide richer and more compelling acoustics treatment. Project Acoustics can model the effect of walls, doors, and other scene geometry on a sound. It's an effective option for cases where the relevant scene geometry is known at development time.
```
Comment (kegodin) [inline mixed-reality-docs/spatial-sound.md]: For 'next steps' at the end of this file, you could link to spatial-sound-in-unity.md and spatial-sound-design.md

File: mixed-reality-docs/voice-input-in-directx.md [hunk 4]
```diff
@@ -129,17 +129,17 @@ From *HolographicVoiceInputSampleMain::Update*:
    }
 ```
 
-## Use dictation for one-shot recognition of speech phrases and sentences
```
Comment (kegodin) [inline mixed-reality-docs/voice-input-in-directx.md]: Above, you can remove the sentence "Make use of the data however applicable to your app scenario."

Comment (jborsecnik) [reply mixed-reality-docs/voice-input-in-directx.md]: Thanks for all the answers. I haven't modified a TOC before. I'll figure out how to do that shortly.

**Comment by kegodin** [comment]
@jessemcculloch Hi Jesse, this PR is ready for merging

**Comment by kegodin** [comment]
#sign-off

**Pull request merged**